NCEP/docs/source/outputs.rst

```diff
@@ -0,0 +1,8 @@
+######################
+Product
+######################
+
+The GraphCastGFS model runs 4 times a day at 00Z, 06Z, 12Z, and 18Z cycles, which both applies on 13 levels and 37-levels. 
+The near real-time forecast outputs along with inputs are available on `AWS <https://noaa-nws-graphcastgfs-pds.s3.amazonaws.com/index.html>`_. For each cycle, the dataset contains input files to feed into GraphCast found in the directory ./graphcastgfs.yyyymmdd/hh/input 
+and 10-day forecast results for the current cycle found iin the directory ./graphcastgfs.yyyymmdd/hh/forecasts_13_levels and 
+./graphcastgfs.yyyymmdd/hh/forecasts_37_levels, respectively. 
```

Comment:
@LinlinCui-NOAA Can you list the 13 and 37 pressure levels in this file? Also the output fields? Thanks

Reply:
@junwang-noaa Ok.

Reply:
@SadeghTabas-NOAA I noticed that there are only 31 levels here:
            variables_to_extract['.pgrb2.0p25.f000'][':SPFH|VVEL|VGRD|UGRD|HGT|TMP:']['levels'] = [':(1|2|3|5|7|10|20|30|50|70|100|150|200|250|300|350|400|450|500|550|600|650|700|750|800|850|900|925|950|975|1000) mb:']